SU-SWS/linky_clicky
# New BAM Feature that works.

# READY FOR REVIEW

# Summary
- The old Backup and Migrate Feature sucked. This one doesn't.

# Needed By (Date)
- Would be nice to have for tomorrow (3.7) for `people1` testing, but we also can just test on this branch.

# Urgency
- Fixes a broken test.

# Steps to Test

1. Start with a fresh install of the `stanford` profile. Ensure that you don't have any stray files in your `sites/default/files/` directory.
2. Run this Feature.
3. See that it passes, _and_ that no backup files are left behind at `admin/config/system/backup_migrate/backups` 

# Affected Projects or Products
- Mostly testing for `Stanford-Drupal-Profile`

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/CI-10
- Other PRs
- Anyone who should be notified? @sherakama, @cjwest, @pookmish, and @boznik might be interested in this.
